### ISSUE TITLE
Minor problem

### ISSUE THREAD

**Issue by voldemortX** [yzou2/CRST#19]
https://github.com/yzou2/CRST/blob/0374db84d2c22fbbab071d2cafe77e144af31d90/crst_seg.py#L663

Division by thresholds first does have a very small chance to change the following argmax results.
e.g. A pixel with 2 classes has softmax result as [0.89, 0.11], but the thresholds so happens to be [0.9, 0.09], then it will satisfy weighted_prob > 1, but it is not even predicted as class 2.
Maybe this is indeed the original intent of the paper, but I'll just point it out here.
I call this minor problem because the example above is very unlikely to happen in actual training.
